package.json

```diff
@@ -35,7 +35,7 @@
     "build:clean": "lerna run build:clean --scope app --scope homepage && rimraf www",
     "build:common": "lerna run build:dev --scope @codesandbox/common --stream",
     "build:components": "lerna run build:dev --scope @codesandbox/components --stream",
-    "build:deps": "lerna run build:dev",
+    "build:deps": "lerna run build:dev --scope @codesandbox/components --stream && lerna run build:dev",
```

Comment:
Do we still need this? Lerna should be able to find the right order and most efficient parallelization of building based on dependencies.